src/ploomber/cli/install.py

```diff
@@ -30,33 +34,43 @@ def main(use_lock):
         it uses regular files and creates the lock ones after installing
         dependencies
     """
+    telemetry.log_api("install-started")
     HAS_CONDA = shutil.which('conda')
     HAS_ENV_YML = Path(_ENV_YML).exists()
     HAS_ENV_LOCK_YML = Path(_ENV_LOCK_YML).exists()
     HAS_REQS_TXT = Path(_REQS_TXT).exists()
     HAS_REQS_LOCK_TXT = Path(_REQS_LOCK_TXT).exists()
 
     if use_lock and not HAS_ENV_LOCK_YML and not HAS_REQS_LOCK_TXT:
-        raise exceptions.ClickException(
-            'Expected and environment.lock.yaml '
-            '(conda) or requirementx.lock.txt (pip) in the current directory. '
-            'Add one of them and try again.')
+        Err = 'Expected and environment.lock.yaml ' \
```

Comment:
can we go back to the old formatting? I find parenthesis more readable than `\`, also upper case variables are reserved for classes, it's better to use `err`

Reply:
Sure